### DISCUSSION TITLE
Drop Prometheus version v2.4.3 in circle CI test?

### DISCUSSION THREAD

**Discussion by yeya24** [thanos-io/thanos#4020] [General]
Hi, I am thinking about dropping Prometheus v2.4.3 in our circle CI test.

The versions are configured in https://github.com/thanos-io/thanos/blob/main/.bingo/variables.env#L41.

~~When I am working on #4013  and adding `--web.enable-lifecycle` to Prometheus to support hot reload, I found that the v2.4.3~~ ~~version doesn't even support it. v2.4.3 is old and I think we can replace it with a newer version.~~ 

As we introduce some new features (exemplars, metadata, etc) which require higher Prometheus version support recently, I think it is time to replace it with Prometheus version v2.26.0.

**Comment by bwplotka** [comment]
Well not really. There are a couple of versions. This is essentially for compatibility testing. We sometimes run tests for multiple Prometheus instances. What do you want to achieve? 🤗

**Reply by yeya24** [reply]
Yes, we can still keep it. Sorry for the issue.
Then what about adding the latest Prometheus version to the CI?

**Reply by bwplotka** [reply]
Yes 10000%! Thanks
